WordPressKit/WordPressComRestApi.swift

```diff
@@ -421,6 +422,100 @@ open class WordPressComRestApi: NSObject {
         let fileURL = URL(fileURLWithPath: NSTemporaryDirectory()).appendingPathComponent(fileName)
         return fileURL
     }
+
+    // MARK: - Async
+
+    private lazy var urlSession: URLSession! = {
+        let configuration = URLSessionConfiguration.default
+
+        var additionalHeaders: [String: AnyObject] = [:]
+        if let oAuthToken = self.oAuthToken {
+            additionalHeaders["Authorization"] = "Bearer \(oAuthToken)" as AnyObject?
+        }
+        if let userAgent = self.userAgent {
+            additionalHeaders["User-Agent"] = userAgent as AnyObject?
```

Comment:
Since we know from the `if let` succeeding into this code path that the two values are not nil, do we need to cast to `AnyObject?` or can we use drop nullability with `AnyObject`?

```suggestion
            additionalHeaders["Authorization"] = "Bearer \(oAuthToken)" as AnyObject
        }
        if let userAgent = self.userAgent {
            additionalHeaders["User-Agent"] = userAgent as AnyObject
```